import-service/serverless.yml

```diff
@@ -0,0 +1,36 @@
+service: rsschool-node-aws-s3-task5
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+
+  iamRoleStatements:
+    - Effect: "Allow"
+      Action:
+        - "s3:ListBucket"
+        - "s3:*"
+      Resource:
+        - arn:aws:s3:::rsschool-node-aws-s3-task5
+        - arn:aws:s3:::rsschool-node-aws-s3-task5/*
+
+functions:
+  importProductsFile:
+    handler: importProductsFile/handler.importProductsFile
+    events:
+      - httpApi:
+          path: /import
+          method: get
+          request:
+            parameters:
+              querystrings:
+                name: ""
```

Comment:
https://www.serverless.com/framework/docs/providers/aws/events/apigateway/#request-parameters
true/false in documentation

Reply:
Good catch!